rr/doc/readme_rr.html

```diff
@@ -767,7 +771,7 @@ <h4>Credits</h4>
     <br />
     <p class="faq-q"><a id="Translation_team" />Translation team:</p>
     <ul class="version-list">
-      <li>French Translation: Mornagest, Thot, Mathrim Cauthon, Lothringen, Ma�l and Graoumf (of the d'Oghmatiques)</li>
+      <li>French Translation: Mornagest, Thot, Mathrim Cauthon, Lothringen, Ma�l and Graoumf (of the d'Oghmatiques)</li>
```

Comment:
Saved with the wrong encoding.

Reply:
What's wrong with the harp in 2.6.5.0?

Reply:
Okay, I see. The warning is harmless but I've written a revised patch to accommodate this most recent version.

Thank you for your contribution, but I cannot merge your commit.
- The already mentioned encoding issue.
- Commenting out the function call is an inappropriate way to address the issue.
- Feature/fix and version bump should be at least 2 different commits.